scripts/eurostat/health_determinants/alcohol_consumption/input_files.py

```diff
@@ -0,0 +1,68 @@
+# Copyright 2022 Google LLC
```

Comment:
Rename this module to download_input_files

Reply:
Changed the module name.